src/resources/views/volumesSidebar.blade.php

```diff
@@ -1,3 +1,3 @@
 @if (($user->can('edit-in', $volume) || $user->can('sudo')) && $volume->isImageVolume() && !$volume->hasTiledImages())
-    <sidebar-tab name="maia" icon="robot" title="Perform Machine Learning Assisted Image Annotation (MAIA)" href="{{route('volumes-maia', $volume->id)}}"></sidebar-tab>
+    <sidebar-tab name="ptp" icon="circle" title="Perform Point to Polygon conversion" href="{{route('volumes-ptp-conversion', $volume->id)}}"></sidebar-tab>
```

Comment:
I thought that maybe we could use the same "magic hat" icon than MagicSAM, as the feature is very similar. It's basically MagicSAM on a volume level. @dlangenk what do you think?

Reply:
Yeah it would be nice to use the same logo.